test/functionalTest/cases/0350_onanychange_subscriptions/onanychange_empty_cond_values.test

```diff
@@ -0,0 +1,422 @@
+# Copyright 2016 Telefonica Investigacion y Desarrollo, S.A.U
```

Comment:
The 4 tests included in this PR are almost identical. Thus, you can review just one and check in the others just the following bits:

* brokerStart  line uses -noCache in the proper files
* That condValues is `[]` or has been omitted in the proper files.